reddit_liveupdate/validators.py

```diff
@@ -28,6 +30,19 @@ def run(self, id):
             return None
 
 
+class VLiveUpdateEventUrl(VLiveUpdateEvent):
+    def run(self, url):
+        if not url:
+            return None
+
+        u = UrlParser(url)
+        id = re.match(r'/live/(\w+)/?', u.path)
```

Comment:
nit: `id` is a keyword in python so I tend to try to use other variable names when possible.